static/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -167,12 +169,19 @@ class AccountNotificationFineTuning extends AsyncView<Props, State> {
 
   renderBody() {
     const {params} = this.props;
-    const {fineTuneType} = params;
+    const {fineTuneType: param} = params;
+    const fineTuneType = getNotificationTypeFromPathname(param);
 
     if (
-      ['alerts', 'deploy', 'workflow', 'activeRelease', 'approval', 'quota'].includes(
-        fineTuneType
-      )
+      [
+        'alerts',
+        'deploy',
+        'workflow',
+        'activeRelease',
+        'approval',
+        'quota',
+        'spikeProtection',
+      ].includes(fineTuneType)
```

Comment:
Now is probably a good time to pull this into a constant at the top of the file.